tools/scripts-utils/src/path.js

```diff
@@ -23,22 +23,10 @@ function resolveBin(modName, { executable = modName, cwd = process.cwd() } = {})
 	} catch (_error) {
 		// ignore _error
 	}
-	try {
-		const modPkgPath = require.resolve(`${modName}/package.json`);
-		const modPkgDir = path.dirname(modPkgPath);
-		const mod = JSON.parse(fs.readFileSync(modPkgPath));
-		const { bin } = mod.bin;
-		const binPath = typeof bin === 'string' ? bin : bin[executable];
-		const fullPathToBin = path.join(modPkgDir, binPath);
-		if (fullPathToBin === systemCommandPath) {
-			return executable;
-		}
-		return fullPathToBin.replace(cwd, '.');
-	} catch (error) {
-		if (systemCommandPath) {
-			return executable;
-		}
-		throw error;
+	if (process.platform === 'win32' && systemCommandPath) {
+		return systemCommandPath;
```

Comment:
cross-spawn was doing this in case of ENOENT (bin not found).
as which command return us the full path (so with `.CMD` at the end ie (node_modules\\.bin\\webpack.CMD) we rely on this now